Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'config' # Settings to manage configs on different instances
 gem 'connection_pool' # Used for redis
 gem 'csv' # will be removed from standard library in Ruby 3.4
 gem 'dor-event-client'
-gem 'dor-workflow-client' # audit errors are reported to the workflow service
+gem 'dor-services-client', '~> 15.0'
```

Comment:
Note: I had to pin this as it was stuck on 8.0 for some reason, that may no longer be an issue with dor-workflow-client removed. I will check asap, but I don't feel like this should be a blocker at this time.